code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -318,7 +318,7 @@
 	var/stim_energy_change
 	stim_energy_change =heat_scale + (STIMULUM_FIRST_RISE(heat_scale**2)) - (STIMULUM_FIRST_DROP(heat_scale**3)) + (STIMULUM_SECOND_RISE(heat_scale**4)) - (STIMULUM_ABSOLUTE_DROP(heat_scale**5))
 
-	ASSERT_GAS(/datum/gas/stimulum,air)
+	ASSERT_GAS(/datum/gas/stimulum, air)
```

Comment:
what's with the space here

Reply:
I didn't revert the whitespace changes and I'm not going to.

Reply:
there's not meant to be a space there

Reply:
it shouldn't make a difference

Reply:
"Shouldn't" as a key word

Reply:
better safe than sorry

Reply:
It doesn't make difference